lib/h.js

```diff
@@ -4,6 +4,10 @@ const flatten = require('lodash.flattendeep');
 const VNode = require('./vnode');
 
 module.exports = (component, props, ...children) => {
+	if (typeof component !== 'function' && typeof component !== 'string') {
+		throw new TypeError(`ink.h: expected component to be a function, but received "${component}". You may have forgotten to export a component.`);
```

Comment:
Could you remove `ink.h` and start with `Expected`?

Also, replace `"${component}"` with `typeof component`.